suites/create.js

```diff
@@ -420,6 +403,19 @@ export function runDataIntegrityProofFormatTests({
         }
       });
     }
+    if(optionalTests.proofChain) {
+      it('If "proof.previousProof" property exists, it MUST be a string ' +
+        'value or unordered list of string values.', function() {
+        for(const proof of proofs) {
+          if(proof.previousProof) {
```

Comment:
would this raise an error if previousProof doesn't exist?

Reply:
no but the statement as says `If "proof.previousProof" property exists` is your point here that that can safely not be an optionalTest?

Reply:
Addressed here: https://github.com/w3c-ccg/data-integrity-test-suite-assertion/pull/110/commits/c6c5810f0f394fc87d5d13ed88e89b6b3dd6f815